src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -12134,6 +12135,88 @@ public void StrongNameProviderWithCustomTempPath()
             Assert.True(!comp.SignUsingBuilder);
         }
 
+        [Theory]
+        [InlineData(@"/features:InterceptorsPreviewNamespaces=NS1.NS2;NS3.NS4")]
+        [InlineData(@"/features:""InterceptorsPreviewNamespaces=NS1.NS2;NS3.NS4""")]
```

Comment:
Let's also test with various invalid strings, such as `NS1..NS2`, `.`, or characters that don't belong in a namespace. It looks like you prefer to just silently ignore those (they won't have any effect).
Also, are we case-sensitive on "InterceptorsPreviewNamespaces" portion or not? (it looks like we are, is that intentional?)

Reply:
Feature names are generally case-sensitive. I'll test when the feature is passed multiple times. Though we expect our build task to construct this string, we don't expect a project to include `<Features>InterceptorsPreviewNamespaces=...</Features>` in their project directly.

Reply:
Have added the suggested tests either here or in InterceptorsTests.

Reply:
• Do we have a test with `InterceptorsPreviewNamespaces` passed in multiple times?

Reply:
See `FeaturesInterceptorsPreviewNamespaces_Duplicate`

Reply:
Apparently I wrote but did not commit this

Reply:
Did you forget to push a commit? I can't see that test :-P